modules/data-storage-management.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/optimizing-storage.adoc
+
+[id="data-storage-management_{context}"]
+= Data storage management
+
+The following table summarizes the main directories that {product-title} components write data to.
+
+.Main directories for storing {product-title} data
+[options="header,footer"]
+|===
+|Directory|Notes|Sizing|Expected growth
+
+|*_/var/lib/etcd_*
+|Used for etcd storage when in Multi-Master mode, when etcd is made standalone by an administrator, or when storing the database.
+|Less than 20 GB.
+
+Database can grow up to 8 GB.
+|Will grow slowly with the environment. Only storing metadata.
+
+Additional 20-25 GB for every additional 8 GB of memory.
+
+|*_/var/lib/containers_*
+|This is the mount point for the CRI-O runtime. Storage used for active container runtimes (including Pods) and storage of local images (not used for registry storage).
+|50 GB for a Node with 16 GB memory.
```

Comment:
this can be misleading I think it's important that we designate the information here should not be used to determine the minimum requirements for the cluster. For example, Azure currently will require a 1TB osDisk to meet minimum requirements. This is due to the nature of cloud providers' storage offerings.

Reply:
I just don't want to get a bug where someone created an Azure cluster with 100GB osDisk and link this back as the reference they used for that decision.

Reply:
Thanks, @hexfusion - I've added, _Note that this sizing should not be used to determine minimum cluster requirements._

Reply:
I appreciate that, thank you.